tests/functional/firefox/test_mobile.py

```diff
@@ -12,8 +12,6 @@
 @pytest.mark.nondestructive
 def test_get_firefox_send_to_device_success(base_url, selenium):
     page = FirefoxMobilePage(selenium, base_url).open()
-    modal = page.click_get_firefox_button()
```

Comment:
It's been a while since these tests were run, so I'd suggest pushing this to `run-integration-tests` again to make sure things pass.

Reply:
- pipeline has passed
- fixed the missing ftl file for the exp template
- fixed typo in exp static file.